lib/taskana-core/src/main/java/pro/taskana/impl/ClassificationServiceImpl.java

```diff
@@ -351,4 +352,78 @@ public Classification getClassificationByTask(Task task) throws ClassificationNo
         }
     }
 
+    @Override
+    public void deleteClassification(String classificationKey, String... domains) throws ClassificationInUseException, NotAuthorizedException, ClassificationNotFoundException {
+        if (domains.length == 0) {
```

Comment:
This can throw a NullPointer, because String.../String[] can be given as NULL. Please change to:
_( ==null || length==0)_